src/Api.php

```diff
@@ -232,4 +242,20 @@ public function json(ResponseInterface $response) {
     return json_decode($response->getBody());
   }
 
+  public function sendWebHook($message)
+  {
+    $text =  $message;
+    $headers = [
+      'Content-type: application/json'
+    ];
+
+    $body = [
+      "text"  =>  $text,
+      "username" => "fastly-drupal-bot",
+      "icon_emoji"=> ":drupal:"
+    ];
+
+    return $this->httpClient->post($this->webHookURL, array ("headers" => $headers, "json" => $body));
```

Comment:
Notice this will add a external call from PHP and a potential performance hit depending on Slack's service latency.